src/group_gui.cpp

```diff
@@ -369,6 +369,7 @@ class VehicleGroupWindow : public BaseVehicleListWindow {
 		this->groups.ForceRebuild();
 		this->groups.NeedResort();
 		this->BuildGroupList(vli.company);
+		this->group_sb->SetCount((uint)this->groups.size());
```

Comment:
SetCount should probably actually take a uint, but currently it takes an int, so I'm not sure what this cast is doing

Reply:
IIRC I copied that line from here, so that's why it's there :)
https://github.com/OpenTTD/OpenTTD/blob/40ebf3c7659201b5365839b1371c4f9e84feea50/src/group_gui.cpp#L515

Reply:
Weird. Don't worry about it then, it can be fixed later (when SetCount is actually converted to a uint)